chromadb/db/clickhouse.py

```diff
@@ -121,10 +121,7 @@ def _create_where_clause(
     #
     def create_collection(
         self, name: str, metadata: Optional[Dict] = None, get_or_create: bool = False
-    ):
-        if metadata is None:
```

Comment:
We want the metadata to use None as a the null-signifier not the empty dict. json.dumps() and loads() properly handles None so we can remove this.